test/e2e/network/service.go

```diff
@@ -550,16 +553,17 @@ func continuousEcho(host string, port int, timeout time.Duration, maxAttempts in
 	numErrors := 0
 
 	for i := 0; i < maxAttempts; i++ {
-		if i == 4 {
-			framework.Logf("Continuous echo waiting for signal to continue")
+		time.Sleep(1 * time.Second)
+		// wait for a signal to continue if we reach the thresold
+		if i == thresold {
```

Comment:
thresold and number of allowed errors is the same, this way if we fail the first tries we fail the test